packages/data/tests/unit/services/navi-facts-elide-test.js

```diff
@@ -0,0 +1,68 @@
+import { module, test, skip } from 'qunit';
+import { setupTest } from 'ember-qunit';
+import { setupMirage } from 'ember-cli-mirage/test-support';
+import GraphQLScenario from 'dummy/mirage/scenarios/graphql';
+
+const TestRequest = {
+  logicalTable: {
+    table: 'table1',
+    timeGrain: 'grain1'
+  },
+  metrics: [{ metric: 'metric1' }, { metric: 'metric2' }],
+  dimensions: [{ dimension: 'dimension1' }, { dimension: 'dimension2' }],
+  filters: [
+    { dimension: 'dimension3', operator: 'in', values: ['v1', 'v2'] },
+    {
+      dimension: 'dimension4',
+      operator: 'in',
+      values: ['v3', 'v4']
+    }
+  ],
+  having: [
+    {
+      metric: 'metric1',
+      operator: 'gt',
+      value: 0 // TODO switch to `values: [0]` after `value` backwards compatibility has been removed
+    }
+  ],
+  intervals: [
+    {
+      start: '2015-01-03',
+      end: '2015-01-04'
+    }
+  ]
+};
+
+module('Unit | Service | Navi Facts - Elide', function(hooks) {
+  setupTest(hooks);
+  setupMirage(hooks);
+
+  hooks.beforeEach(function() {
+    this.service = this.owner.lookup('service:navi-facts');
+    GraphQLScenario(this.server);
+  });
+
+  test('fetch', async function(assert) {
+    assert.expect(2);
+
+    const response = await this.service.fetch(TestRequest);
```

Comment:
Can we do a deep equals assert on the response directly?

Reply:
No because the values in the response are generated by faker

Reply:
Can we set a faker seed with a base64 string of the query https://faker.readthedocs.io/en/master/#seeding-the-generator

Reply:
Faker requires the seed to be a base 10 int, so I just seeded it off the total length of the table name, args, and fields in the request